src/routes/barcharts.svelte

```diff
@@ -19,77 +19,69 @@
+			/>
 			<Subheading text="(% of individuals who used internet within the last 3 months)" />
+			<Chart dataset={data} {onSelect} {selected} dataPoint="have_cookies_knowledge_perc" />
+			<Title
+				text="Individuals have ever changed the settings in their internet browser to prevent or limit cookies on any of their devices."
+			/>
 			<Subheading text="(% of individuals who used internet within the last 3 months)" />
-		</div>
-		<div class="chart-row">
-			<Chart
-				dataset={data}
-				{onSelect}
-				{selected}
-				dataPoint="have_cookies_knowledge_perc" />
-			<Chart
-				dataset={data}
-				{onSelect}
-				{selected}
-				dataPoint="change_default_settings_perc"/>
-		</div>
-		<div class="title-row"> 
-			<Title text="Individuals use software that limits the ability to track their activities on the internet." />
-			<Title text="Individuals manage access to personal data on the internet." />
-		</div>
-		<div class="subheading-row">
+			<Chart dataset={data} {onSelect} {selected} dataPoint="change_default_settings_perc" />
+
+			<Title
+				text="Individuals use software that limits the ability to track their activities on the internet."
+			/>
 			<Subheading text="(% of individuals who used internet within the last 3 months)" />
+			<Chart dataset={data} {onSelect} {selected} dataPoint="use_restrictive_software_perc" />
+
+			<Title text="Individuals manage access to personal data on the internet." />
 			<Subheading text="(% of individuals who used internet within the last 3 months)" />
-		</div>
-		<div class="chart-row">
-			<Chart
-				dataset={data}
-				{onSelect}
-				{selected}
-				dataPoint="use_restrictive_software_perc"/>
-			<Chart
-				dataset={data}
-				{onSelect}
-				{selected}
-				dataPoint="manage_personal_data_perc" />
-		</div>
+			<Chart dataset={data} {onSelect} {selected} dataPoint="manage_personal_data_perc" />
 		{/await}
 	</section>
 	<section class="footer">
-		<Source/>
+		<Source />
 	</section>
-	
 </main>
 
 <style>
+	/* CSS Reset: https://www.joshwcomeau.com/css/custom-css-reset/ */
+	:global(*) {
+		margin: 0;
+	}
+
+	:global(p) {
+		margin-bottom: 1rem;
+	}
+
+	:global(body) {
+		margin: 1rem;
+	}
+	/* End of CSS reset */
+
 	@import url('https://rsms.me/inter/inter.css');
-	
+
 	main {
-		font-family: 'Inter', sans-serif; 
-		display: flex;
-    	flex-direction: column;
+		font-family: 'Inter', sans-serif;
+		font-size: 16px;
 	}
-	
-	.chart-container {
-		display: flex;
-		flex-direction: column;
+
+	@media screen and (max-width: 1000px) {
+		main {
+			font-size: 12px;
+		}
 	}
-	
-	.title-row, .subheading-row, .chart-row {
-		display: flex;
-		flex-direction: row;
-		gap: 20px;
-		width: 100%;
+
+	.chart-container {
+		display: grid;
+		grid-template-columns: 1fr 1fr;
+		grid-template-rows: min-content min-content 1fr min-content min-content 1fr;
+		grid-column-gap: 20px;
+		grid-row-gap: 0;
+		grid-auto-flow: column;
```

Comment:
This property lets us use order of elements as their placement. So we go with each element and position in first empty space by column then by row. For more specific placement we can use [grid-area](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-area) and similar